ndk/src/events/gift-wrapping.ts

```diff
@@ -0,0 +1,117 @@
+import { NDKEvent, NostrEvent } from "./index.js";
+import { NDKPrivateKeySigner } from "../signers/private-key";
+import { VerifiedEvent, getEventHash, verifiedSymbol } from "nostr-tools";
+import { EncryptionNip } from "./encryption.js";
+import { NDKUser } from "../user/index.js";
+import { NDKSigner } from "../signers/index.js";
+import { NDKKind } from "./kinds/index.js";
+
+// NIP 59 - adapted from Coracle
+
+export type GiftWrapParams = {
+    encryptionNip?: EncryptionNip
+    rumorKind?: number;
+    wrapKind?: 1059 | 1060
```

Comment:
What is a `1060`? Never seen that.

Reply:
This is a takeover from @manimejia and I didn't know neither what for it is. And this kind number isn't specified in the nips. I'll remove the field from this type.